File: src/examples/python/trackers/create_tracker.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_TEST_API_KEY')

tracker = easypost.Tracker.create(tracking_code='EZ1000000001', carrier='USPS')

print(tracker)

File: src/examples/node/trackers/create-tracker.js
const Easypost = require('@easypost/api');
const dotenv = require('dotenv');

dotenv.config({ path: '/Users/jhammond/git/easypost/easypost-tools/.env' });
const api = new Easypost(process.env.EASYPOST_TEST_API_KEY);

const tracker = new api.Tracker({
  tracking_code: '9400110898825022579493',
  carrier: 'USPS',
});

tracker.save().then(console.log);
